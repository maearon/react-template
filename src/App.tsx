import ValueStepper from "./components/ValueStepper/ValueStepper";

const App = () => {
  return (
    <div className="w-screen h-screen bg-neutral-900 text-neutral-100 flex items-center justify-center px-4">
      <div className="w-full max-w-md space-y-4">
        <label className="text-sm font-semibold text-neutral-300">Unit value</label>
        <div className="w-full bg-neutral-950 p-4 rounded-lg space-y-4">
          <ValueStepper />
        </div>
      </div>
    </div>
  );
};

export default App;