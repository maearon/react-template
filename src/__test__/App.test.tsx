import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import App from "../App";

test('renders App container', () => {
  const { container } = render(<App />);
  expect(container).toBeTruthy();
});
