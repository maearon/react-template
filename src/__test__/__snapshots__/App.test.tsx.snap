// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`toUpperCase 1`] = `
<DocumentFragment>
  <div
    class="w-screen h-screen bg-neutral-950 flex items-center justify-center text-neutral-100"
  >
    <div
      class="w-96 bg-neutral-800 p-4 rounded-lg"
    >
      <div
        class="p-4 flex flex-col items-start gap-2"
      >
        <div
          class="flex gap-2"
        >
          <button
            class="px-3 py-1 border rounded bg-blue-500 text-white"
          >
            %
          </button>
          <button
            class="px-3 py-1 border rounded bg-white text-black"
          >
            px
          </button>
        </div>
        <div
          class="flex gap-2 items-center"
        >
          <button>
            âˆ’
          </button>
          <input
            class="border p-1 w-24 text-center"
            value="1"
          />
          <button>
            +
          </button>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
